src/tokens/colors.js

```diff
@@ -10,6 +10,7 @@ import type {ColorTokensT} from './types.js';
 const colors: ColorTokensT = {
```

Comment:
Any reason why we're keeping the tokens folder at all? The tokens are part of the theming, so it seemed out of place that some items would be in tokens and some in themes. Also, this particular file contains the primitives and are not tokens. Considering the refactor going on in this PR, it might be worth us better organizing and naming some of these files.